api/lib/application/admin-members/index.js

```diff
@@ -21,6 +23,39 @@ exports.register = async function (server) {
         tags: ['api', 'admin-members'],
       },
     },
+    {
+      method: 'PATCH',
```

Comment:
On aurait pu aussi faire un PUT sur `/api/admin/admin-members/{id}/role`
cf https://blog.octo.com/should-i-put-or-should-i-patch/ et les routes qu'on a implémenté récemment